worker/transfer.go

```diff
@@ -156,7 +162,9 @@ func parallelUploadSlab(ctx context.Context, sp storeProvider, shards [][]byte,
 	}
 	if rem > 0 {
 		if errs == nil {
-			return nil, nil, fmt.Errorf("rem > 0 (%v) but errs is nil - this should not happen", rem)
+			err := fmt.Errorf("rem > 0 (%v) but errs is nil - this should not happen", rem)
+			logger.Errorf("upload failed, err: %v", err)
```

Comment:
Shouldn't this be in `uploadSlab`? Because if `errs != nil` the upload also failed.